pom.xml

```diff
@@ -1512,7 +1512,7 @@
             <dependency>
                 <groupId>com.fasterxml.jackson.core</groupId>
                 <artifactId>jackson-databind</artifactId>
-                <version>${jackson.version}</version>
+                <version>2.8.11.1</version> <!-- TODO sync. with jackson.version -->
```

Comment:
Before merging, this TODO needs cleaning up (I'm assuming that's why this is still flagged as a WIP).  Currently we seem to have two versions of `jackson` (2.8.11 in `jackson.version` and 2.8.11.1 here).  It'd be nice to get this set in one place and inherited everywhere else.

Reply:
No, the TODO is for later, when it is *possible* to sync. them.  There is no jackson-core or jackson-annotations 2.8.11.1, but jackson-databind 2.8.11 still has the bug.

It's WIP because I'm still looking into fixing some of the remaining Snyk complaints.  Sorry, should have said so.